### PR TITLE
Move dashboard right column elements into a hook

### LINKED ISSUE #25030: Move dashboard right column elements into MBO
#### Is your feature request related to a problem?

These elements have to be moved outside the Core.

image
#### Alternatives you've considered

Have to be move into the mbo module

### PR DESCRIPTION
**Pull request by intraordinaire**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Replace data and links to addons on the Dashboard by hook that can display a third column.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #25030 
| How to test?      | In the dashboard, right column and link at the bottom of the middle column should be deleted. If `ps_mbo` of this [PR](https://github.com/PrestaShopCorp/ps_mbo/pull/114) is installed, they must be back again.
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26239)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

File: controllers/admin/AdminDashboardController.php
Comment (matks) [inline controllers/admin/AdminDashboardController.php:307]: ping @PrestaShop/prestashop-maintainers we'll put another link when we have our own API 😄

**Review by matks — DISMISSED** [review]
Looks good to me 😄  bravo !

Did you subscribe on https://hacktoberfest.digitalocean.com/ ? this would be marked as a valid contribution

**Review by PierreRambaud — DISMISSED** [review]

**Comment by tegbessou** [comment]
I can't approve, but I don't see any problem with this PR.

**Review by PierreRambaud — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Comment by intraordinaire** [comment]
Hello @matks & @PierreRambaud
I've resolved a merge conflict with the `hook.xml` file, can you check this again ? :)

**Review by PierreRambaud — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]
@intraordinaire Thank you for your contribution. Could you add in an `autoupgrade` PR the new hook for migration SQL ?

**Review by PierreRambaud — APPROVED** [review]

**Comment by intraordinaire** [comment]
> @intraordinaire Thank you for your contribution. Could you add in an `autoupgrade` PR the new hook for migration SQL ?

@Progi1984 [here it is](https://github.com/PrestaShop/autoupgrade/pull/429)

**Comment by marwachelly** [comment]
Hello, 
PR is tested , In the dashboard, right column and link at the bottom of the middle column is deleted. when ps_mbo of this PR is installed, they are back again. But to be sure that the changes are saved after an upgrade, this PR is still blocked by this issue https://github.com/PrestaShop/PrestaShop/issues/26503
Thanks,

**Comment by PierreRambaud** [comment]
> Hello, PR is tested , In the dashboard, right column and link at the bottom of the middle column is deleted. when ps_mbo of this PR is installed, they are back again. But to be sure that the changes are saved after an upgrade, this PR is still blocked by this issue #26503 Thanks,

I think the PR can be validated without waiting for the autoupgrade issue, it's not related imho

**Comment by Robin-Fischer-PS** [comment]
Hi,

Since the upgrade issue might not be fixed soon, I suggest we validate this PR without waiting. To do so, could a @PrestaShop/prestashop-core-developers launch again the tests on this PR ? Some of them are KO.

Thanks :)

**Comment by Progi1984** [comment]
@Robin-Fischer-PS It's good : CI is :ok:

**Comment by Robin-Fischer-PS** [comment]
I agree with the really good suggestion of @Robin-Fischer-PS , so it's QA :heavy_check_mark: \o/

**Comment by PierreRambaud** [comment]
Thank you @intraordinaire

File: controllers/admin/AdminDashboardController.php
Comment (kpodemski) [reply controllers/admin/AdminDashboardController.php:307]: @matks this is still a good idea Mathieu :D

**Pull request merged**
